server/enums.py

```diff
@@ -1,21 +1,10 @@
-from email.mime import application
 import enum
 
 #starter for db models. feel reorganize or remove once/if internal classses for these are implemented
 
-
-class ImageContentType(enum.Enum):
-    png = "image/png;base64"
-    jpeg = "image/jpeg;base64"
-
-class TextContentType(enum.Enum):
-    markdown = "text/markdown"
-    plain = "text/plain"
-    #application = "application/base64"
-
 class ContentType(enum.Enum):
-    markdown = TextContentType.markdown
-    plain = TextContentType.plain
+    markdown ="text/markdown"
+    plain = "text/plain"
     #application = "application/base64"
-    png = ImageContentType.png
-    jpeg = ImageContentType.jpeg
\ No newline at end of file
+    png = "image/png;base64"
+    jpeg = "image/jpeg;base64"
```

Comment:
Why did you revert back to this?

Reply:
I changed it back because post was changed. Once I got to trying to create posts I realized there was no way to relate the 2 the text/image posts. For example make a text post. Then once you need to make an image, based on the api spec how is one to relate an image post to an already created text post? I know our old model had them related through the post class but from the api side how would you even relate the 2? 

see: https://eclass.srv.ualberta.ca/mod/forum/discuss.php?d=1947466#p5086761